connect/json/src/test/java/org/apache/kafka/connect/json/JsonConverterTest.java

```diff
@@ -301,6 +301,15 @@ public void timestampToConnectOptional() {
         assertNull(schemaAndValue.value());
     }
 
+    @Test
+    public void timestampToConnectDefval() {
```

Comment:
@shikhar Is there a downside to covering the default value case for the 3 other logical types? Seems like these tests are cheap and protect us in the future from any change in logic which may or may not be type-specific.